packages/php82.rb

```diff
@@ -3,49 +3,79 @@
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/php82/8.2.5_armv7l/php82-8.2.5-chromeos-armv7l.tar.zst',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/php82/8.2.5_i686/php82-8.2.5-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/php82/8.2.5_x86_64/php82-8.2.5-chromeos-x86_64.tar.zst'
   })
   binary_sha256({
-    aarch64: '7582d0d7d93047d68649a052bb97aa95eff61260935bd944860452a463022331',
-     armv7l: '7582d0d7d93047d68649a052bb97aa95eff61260935bd944860452a463022331',
-       i686: '08dd37cb428971944008036775bf70fed358902d6b56890bb2411e971533f213',
-     x86_64: '9081724ee645e4c0d8bf5f35ac9711bac3362ad8344705b60dca98aa22be1457'
+    aarch64: 'b7a2262392a2ec627f3630b40726e56512a4c70685d25117ec91a8d7481185e8',
+     armv7l: 'b7a2262392a2ec627f3630b40726e56512a4c70685d25117ec91a8d7481185e8',
+       i686: '117284d2577dde1ff96013aca7c9150a319954ae7d3ecce20c6a502340eaf0b7',
+     x86_64: '5437cdc57c0a93a101e422d388a09dc15b85d8b7105235626f6154c047a5802b'
   })
 
   depends_on 'aspell_en'
-  depends_on 'curl'
-  depends_on 'libgcrypt'
-  depends_on 'libjpeg'
-  depends_on 'libpng'
-  depends_on 'libsodium'
-  depends_on 'libxpm'
-  depends_on 'libxslt'
-  depends_on 'libzip'
+  depends_on 'aspell' # R
+  depends_on 'brotli' # R
+  depends_on 'bz2' # R
+  depends_on 'c_ares' # R
+  depends_on 'curl' # R
+  depends_on 'e2fsprogs' # R
   depends_on 'exif'
   depends_on 'freetds'
-  depends_on 'freetype'
+  depends_on 'freetype' # R
+  depends_on 'gcc' # R
+  depends_on 'gdbm' # R
+  depends_on 'glibc' # R
+  depends_on 'gmp' # R
   depends_on 'graphite'
-  depends_on 're2c'
-  depends_on 'tidy'
-  depends_on 'unixodbc'
-  depends_on 'oniguruma'
+  depends_on 'icu4c' # R
+  depends_on 'krb5' # R
+  depends_on 'libcyrussasl' # R
+  depends_on 'libedit' # R
+  depends_on 'libffi' # R
+  depends_on 'libgcrypt' # R
+  depends_on 'libgpgerror' # R
+  depends_on 'libidn2' # R
+  depends_on 'libjpeg' # R
+  depends_on 'libnghttp2' # R
+  depends_on 'libpng' # R
+  depends_on 'libpsl' # R
+  depends_on 'libsodium' # R
+  depends_on 'libssh' # R
+  depends_on 'libtool' # R
+  depends_on 'libunistring' # R
+  depends_on 'libxml2' # R
+  depends_on 'libxpm'
+  depends_on 'libxslt' # R
+  depends_on 'libzip' # R
+  depends_on 'ncurses' # R
+  depends_on 'oniguruma' # R
+  depends_on 'openldap' # R
+  depends_on 'openssl' # R
   depends_on 'py3_pygments'
+  depends_on 're2c'
+  depends_on 'sqlite' # R
+  depends_on 'tidy' # R
+  depends_on 'unixodbc' # R
+  depends_on 'zlibpkg' # R
+  depends_on 'zstd' # R
 
   no_fhs
 
   def self.preflight
     phpver = `php -v 2> /dev/null | head -1 | cut -d' ' -f2`.chomp
-    abort "PHP version #{phpver} already installed.".lightgreen if ARGV[0] != 'reinstall' && @_ver != phpver && !phpver.empty?
+    return unless ARGV[0] != 'reinstall' && @_ver != phpver && !phpver.empty?
```

Comment:
This won't work since `@_ver` was removed?

Reply:
Is this better?